sdks/cpp/src/agones/sdk.cc

```diff
@@ -15,6 +15,7 @@
 #include "agones/sdk.h"
 
 #include <grpcpp/grpcpp.h>
+
```

Comment:
nit: I'm not sure I understand why there is a new blank line here.

Reply:
Ah! Without this line, we get this error:

```
CMake Error at CMakeLists.txt:28 (_message):
  clang-format code style check failed for: sdks/cpp/src/agones/sdk.cc
```